test/parallel/test-child-process-fork-net.js

```diff
@@ -60,9 +61,11 @@ if (process.argv[2] === 'child') {
 
   const child = fork(process.argv[1], ['child']);
 
-  child.on('exit', function() {
+  child.on('exit', common.mustCall(function(code, signal) {
     console.log('CHILD: died');
```

Comment:
Can you remove this `console.log()` since it's now included in the assertion message.